exonum/src/api/node/public/explorer.rs

```diff
@@ -276,6 +293,59 @@ impl ExplorerApi {
             })
     }
 
+    /// Return call status of committed transaction.
```

Comment:
Nit: "Returns" instead of "return" here and in other places.

Reply:
`fn block` and `fn blocks` annotated with "return" too

Reply:
Feel free to change it there as well